src/app/client/src/app/modules/core/components/search/search.component.ts

```diff
@@ -82,6 +82,7 @@ export class SearchComponent implements OnInit {
    */
   onEnter(key) {
     this.key = key;
+    this.queryParam = {};
```

Comment:
confirm with namritha on this

Reply:
confirmed with namrita no filters should be carried to search page.